### PR TITLE
5.2 Branch - Local Docker environment

### PR DESCRIPTION
**Pull request by desrosj**
See https://core.trac.wordpress.org/ticket/48301.

### CODE REVIEW

**Comment by desrosj** [comment]
Test

File: .env [hunk 1]
```diff
@@ -0,0 +1,34 @@
+##
+# Default configuration options for the local dev environment.
+#
+# All of these options can be overridden by setting them as environment variables before starting
+# the environment. You will need to restart your environment when changing any of these.
+##
```
Comment (desrosj) [inline .env]: Testing again!

File: tools/local-env/scripts/start.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const dotenv = require( 'dotenv' );
+const { execSync } = require( 'child_process' );
+
+dotenv.config();
```
Comment (noisysocks) [inline tools/local-env/scripts/start.js]: This is a single comment

File: tools/local-env/scripts/start.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+// Start the local-env containers.
+execSync( 'docker-compose up -d wordpress-develop', { stdio: 'inherit' } );
```
Comment (noisysocks) [inline tools/local-env/scripts/start.js]: This is a comment in a review

**Comment by noisysocks** [comment]
Test test test

**Comment by desrosj** [comment]
This has become difficult to sync. Going to open a fresh PR.

**Pull request closed without merging**